README.rst

```diff
@@ -1013,6 +1013,17 @@ Choose from the list of available rules:
 
   The correct case must be used for standard PHP types in phpdoc.
 
+* **phpdoc_types_order**
+
+  Sorts PHPDoc types.
+
+  Configuration options:
+
+  - ``null_position`` (``'always_first'``, ``'always_last'``, ``'default'``): forces the
```

Comment:
if the fixer is configured to do nothing, then `isCandidate` shall return `false`.
i already started applying this approach.

we could raise a warning on CLI output that some fixer is configured to do nothing, but I am :-1 for raising error.

--------------

@juliendufresne , if `default` means to not move it on top of sorting algorithm, then name is misleading. for me, `default` is meaningful, like `by default we move the null to...`.